spinn_utilities/ordered_set.py

```diff
@@ -89,16 +89,15 @@ def __reversed__(self):
             yield curr.key
             curr = curr.prev_node
 
-    def pop(self, last=True):
+    def pop(self, last=True):  # pylint: disable=arguments-differ
```

Comment:
Should probably have a "pop_first", but understandable given how integrated this is already!